### PR TITLE
fix: Add accept rental transaction to activity

### LINKED ISSUE #1115: The transaction to accept rentals is not shown in the activity page
## 🎉 Description

Whenever a user accepts a rental, the transaction to accept the rental should be shown in the activity page but it's not being shown. Add the mechanism to show this transaction.

* [x] 🐛 This is a bug report.
* [ ] 📈 This is a feature request.

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR adds to the activity the accept rental transaction item that was missing.

Closes #1115

### CODE REVIEW

File: webapp/src/components/ActivityPage/Transaction/Transaction.tsx [hunk 1]
```diff
@@ -421,6 +423,44 @@ const Transaction = (props: Props) => {
...
+                <T
+                  id="transaction.detail.accept_rental"
+                  values={{
+                    name: (
+                      <Link to={locations.manage(contractAddress, tokenId)}>
+                        {nft ? getAssetName(nft) : ''}
+                      </Link>
+                    ),
+                    pricePerDay: (
+                      <Mana network={nft?.network} inline>
```
Comment (fzavalia) [inline webapp/src/components/ActivityPage/Transaction/Transaction.tsx:447]: What happens when network is undefined?

File: webapp/src/components/ActivityPage/Transaction/Transaction.tsx [hunk 2]
```diff
@@ -421,6 +423,44 @@ const Transaction = (props: Props) => {
...
+                        {/* As this there might be already registered transactions and the price information is new, consider it optional */}
+                        {pricePerDay
+                          ? ethers.utils.formatEther(pricePerDay)
```
Comment (fzavalia) [inline webapp/src/components/ActivityPage/Transaction/Transaction.tsx:450]: I assume pricePerDay is in Wei here

Comment (LautaroPetaccio) [reply webapp/src/components/ActivityPage/Transaction/Transaction.tsx:450]: It is, yes.

File: webapp/src/components/ActivityPage/Transaction/Transaction.tsx (hunk 1 above)
Comment (LautaroPetaccio) [reply webapp/src/components/ActivityPage/Transaction/Transaction.tsx:447]: It is defaulted to `Ethereum` which is the same network that is used for the rental procedures.
[Here's the link to the code that does that](https://github.com/decentraland/ui/blob/60448893d0c4df40a3231aeba52ff8494a7a310b/src/components/Mana/Mana.tsx#L17).

**Pull request merged**